generation/maisi/README.md

```diff
@@ -31,6 +36,64 @@ We retrained several state-of-the-art diffusion model-based methods using our da
 
 </div>
 
+| Dataset     | Model           | LPIPS ↓ | SSIM ↑ | PSNR ↑  | GPU ↓  |
+|-------------|-----------------|----------|--------|---------|--------|
+| MSD Task07  | MAIS VAE        | **0.038**| **0.978**|**37.266**| **0h** |
+|             | Dedicated VAE   | 0.047    | 0.971  | 34.750  | 619h   |
+| MSD Task08  | MAIS VAE        | 0.046    | 0.970  | 36.559  | **0h** |
+|             | Dedicated VAE   | **0.041**|**0.973**|**37.110**| 669h   |
+| Brats18     | MAIS VAE        | **0.026**|**0.0977**| **39.003**| **0h** |
+|             | Dedicated VAE   | 0.030    | 0.0975 | 38.971  | 672h   |
+
+**Table 2:** Performance comparison of the `MAIS VAE` model on out-of-distribution datasets (i.e., unseen during MAISI VAE training) versus `Dedicated VAE` models (i.e., train from scratch on in-distribution data). The “GPU” column shows additional GPU hours for training with one 32G V100 GPU. MAISI VAE model achieved comparable results without additional GPU resource expenditure on unseen datasets.
+
+
+## Time Cost and GPU Memory Usage
+
+### Inference Time Cost and GPU Memory Usage
+| `output_size` | `autoencoder_sliding_window_infer_size` | `autoencoder_tp_num_splits` | Peak Memory | DM Time | VAE Time |
+|---------------|:--------------------------------------:|:---------------------------:|:-----------:|:-------:|:--------:|
+| 256x256x128   | >=[64,64,32], not used                 | 2                           | 14G         | 57s     | 1s       |
```

Comment:
What does this not used mean here? And Does `>=[64,64,32]` means any sliding_window_infer_size larger than `64,64,32` will not influence the infer time?

Reply:
Yes

Reply:
I added some description. Thanks